### DISCUSSION TITLE
Change Go to Documentation Url

### DISCUSSION THREAD

**Discussion by elmalakomar** [mia-platform/community#559] [Q&A]
### What product are you having troubles with?

Console

### What Console version are you using?

v13.0.5

### Description

Hi,
How can I change the url of the following two link "go to documentation"?

The links are 
- in the Homepage of the project
<img width="896" alt="Screenshot 2024-10-10 at 10 58 38" src="https://github.com/user-attachments/assets/929f1a46-fc99-407f-8ac2-f5177bf25f67">

- in the Runtime Environments section of the project
<img width="575" alt="Screenshot 2024-10-10 at 10 59 35" src="https://github.com/user-attachments/assets/89a77945-7764-4cb5-b35f-052510e23e72">

In my current configuration they redirect to different url.

Thanks

**Comment by akelity** [comment]
It is a configuration that can be managed by modifying the link from the CMS. 
As of now, there are no other options, but we will make sure to pass on the feedback for further improvements and developments.
